apps/chat/channels.py

```diff
@@ -361,11 +366,7 @@ def _user_gave_consent(self) -> bool:
 
     def _extract_user_query(self) -> str:
         if self.message.content_type == MESSAGE_TYPES.VOICE:
-            try:
```

Comment:
This will be caught by the exception I added in the `_new_user_message` method and ultimately do there what it did here